controllers/controller.js

```diff
@@ -139,13 +139,41 @@ exports.getTodoLists = async function (req, res) {
   }
 };
 
-
 exports.getEditList = async function (req, res) {
   try {
-    const list = await TodoList.findOne({_id: req.params.id})
-    const tasks = await TodoTask.find({id_list: req.params.id})
-    res.status(200).json({list, tasks})
+    const list = await TodoList.findOne({ _id: req.params.id });
+    const tasks = await TodoTask.find({ id_list: req.params.id });
+    res.status(200).json({ list, tasks });
+  } catch (e) {
+    res.status(500).json({ message: "Something went wrong, try again" });
+  }
+};
+
+exports.editList = async function (req, res) {
+  try {
+    const tasks = req.body.tasks;
+    const newList = await TodoList.findOneAndUpdate(
+      { _id: req.body.list._id },
+      {
+        $set: {
+          title: req.body.list.title,
+        },
+      }
+    );
+    const newTasks = tasks.map(async (task) => {
+      await TodoTask.findOneAndUpdate(
+        { _id: task._id },
+        {
+          $set: {
+            text: task.text,
```

Comment:
You have object with similar properties. Don't need destructurization. You can send first object.